contain-intrinsic-size.bs

```diff
@@ -0,0 +1,125 @@
+<pre class='metadata'>
+Title: CSS Contain Intrinsic Size
+Shortname: css-contain-intrinsic-size
+Level: 1
+Status: UD
+Group: csswg
+Work Status: exploring
+URL: https://wicg.github.io/display-locking/contain-intrinsic-size.html
```

Comment:
Can you publish the current snapshot so I can see it rendered?

Reply:
https://vmpstr.github.io/display-locking/contain-intrinsic-size.html